src/assets/Feed.js

```diff
@@ -1,13 +1,59 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
 import { connect } from 'react-redux';
-import { fetchRecent } from '../store/actions';
+import {
+  fetchPostByRoom,
+  fetchPostsAndFlagsByRoom,
+} from '../store/actions/index';
+import { List, Divider, Select } from 'antd';
 
-const FeedContent = () => {
-  return <h2>Feed</h2>;
+import DiscussionCard from './DiscussionCard';
+
+const Feed = (props) => {
+  const { roomID } = useParams();
+
+  useEffect(() => {
+    if (roomID) {
+      if (props.user.role_id < 2) props.fetchPostByRoom(roomID, 1);
+      else props.fetchPostsAndFlagsByRoom(roomID, 1);
```

Comment:
ahh conditional logic ayeeeee. Good catch, but middleware and stuff is cool too echem. (Will be good also for when future cohorts implement the permissions table logic)